reference/camera.md

```diff
@@ -186,6 +186,15 @@ It is also possible to show the camera image in an external window by double-cli
 After doing it, the overlay disappears and a new window pops up.
 Then, after closing the window, the overlay will be automatically restored.
 
+When the [recognition mode](#wb_camera_has_recognition) is enabled and the simulator is paused, yellow rectangles surround the recognized objects.
```

Comment:
The rectangle are always present even in non-pause mode.

Reply:
I would remove the 'yellow' world as by default they are red (see 'frameColor' in https://www.cyberbotics.com/doc/reference/recognition#field-summary).